src/app.py

```diff
@@ -33,10 +33,12 @@
 import importlib
 import ujson
 import os
+import sys
 from argparse import ArgumentParser
 from utils.sqlite_database import Sqlite_Database
 from multi_processing.coordinator import Coordinator
 import multi_processing.batch_size_calc as batch_size_calc
+import utils.config_validator as config_validator
```

Comment:
create a package named validator and put the config validator here

Reply:
Done